panel/layout.py

```diff
@@ -152,23 +158,23 @@ def pop(self, index):
         self.objects = new_objects
```

Comment:
Should we add Layout = Panel for now?  Or is it unlikely that user code would have used Layout by that name?

Reply:
Seems unlikely that any user would have used an abstract baseclass.